fuzzers/005-tilegrid/generate.py

```diff
@@ -18,7 +21,7 @@
     with open(arg) as f:
         line = f.read().strip()
         site = arg[7:-6]
-        frame = int(line[5:5+8], 16)
+        frame = int(line[5:5 + 8], 16)
```

Comment:
I don't think this is PEP 8. See examples at https://www.python.org/dev/peps/pep-0008/#pet-peeves.

There are a few other things related to indices and slices in that PR that is not conforming to PEP 8. (Just saying. I'm not a PEP 8 evangelists, so I'm not saying this should block the PR. All I'm saying is that it's strange to convert PEP 8 code to non-PEP 8  code in a commit that claims to perform the opposite. ;)

Reply:
The changes were generated by the autopep8 tool.  I assume it implements PEP8 correctly but maybe not.